src/components/send/SendHeader.js

```diff
@@ -117,17 +117,15 @@ export default function SendHeader({
   const isPreExistingContact = (contact?.nickname?.length || 0) > 0;
 
   const name =
-    removeFirstEmojiFromString(userWallet?.label || contact?.nickname || nickname) || userWallet?.ens || contact?.ens || recipient;
+    removeFirstEmojiFromString(contact?.nickname || userWallet?.label || nickname) || userWallet?.ens || contact?.ens || recipient;
```

Comment:
if you assigned a nickname to a contact it was never for wallets that had ENS names